plutarch-test/goldens/monadic.api.example.signatory.dev=true.bench.golden

```diff
@@ -1,2 +1,2 @@
-do.succeeds {"exBudgetCPU":8163467,"exBudgetMemory":16009,"scriptSizeBytes":382}
-do.fails {"exBudgetCPU":11192573,"exBudgetMemory":21069,"scriptSizeBytes":378}
\ No newline at end of file
+do.succeeds {"exBudgetCPU":8163467,"exBudgetMemory":16009,"scriptSizeBytes":392}
+do.fails {"exBudgetCPU":11192573,"exBudgetMemory":21069,"scriptSizeBytes":388}
```

Comment:
what happened here?

Reply:
See my comment farther up, it's just because the length of the name changed and hence the debugging information leads to a different scriptSize.